drivers/led_strip/ws2812_spi.c

```diff
@@ -45,6 +45,7 @@ struct ws2812_spi_cfg {
 	const uint8_t *color_mapping;
 	size_t length;
 	uint16_t reset_delay;
+	uint16_t empty_preamble_frames;
```

Comment:
And then it doesn't need to be stored.

Reply:
I am not sure how to avoid storing this because in principle it can vary from instance to instance so it cannot be easily referenced in `ws2812_strip_update_rgb`

Reply:
Well, it is a problem.